source/includes/harvest/_job_openings.md

```diff
@@ -291,10 +292,25 @@ id | The ID of the opening. Note this is the immutable internal id, and not the
 Parameter | Required | Type | Description
 --------- | ----------- | ----------- | -----------
 opening_id | no | string | This is a string that contains an opening_id. This may be a blank string. Changing an opening_id may re-trigger approvals. For approvals to start recruiting, this will reset approvals only if the job is in draft mode. If the job is open for hiring, these approvals will not reset. For official job approvals, this will reset approvals only if the job is open.
-status | no | string | This can be used to set an open opening to closed by using the word "closed".  Status may only be used to set an open opening to closed. It does not accept any other value and cannot be used to re-open a closed opening. If the last opening is closed, it will close the hiring plan.
+status | no | string | Accepts either "open" or "closed". This can be used to close an open opening with the word "closed" or to open a closed opening with the word "open". Official job approvals will be reset when opening a closed opening only if the Permission Policy named "Reopening a job with approval should require reapproval" is toggled on. If the last opening is closed, it will close the hiring plan.
 close_reason_id | no | integer | When closing, you may provide a close_reason_id. Providing a close_reason_id without closing the opening will return an error.
+custom_fields | no | array | Array of custom field objects containing updated values for custom fields on the opening. See the "Custom Field Object Parameters" section below for more details. Note: updating a custom field may reset official job approvals if this option is selected in the custom field settings page.
 
-**Note**: Only open openings may be patched. Attempting to patch a closed opening will return an error. If the job is closed at the same time the opening_id is changed, approvals will be ignored in favor of closing the opening.
+**Note**: Closed openings must first be reopened before they can be patched by this endpoint. If the job is closed at the same time the opening_id is changed, approvals will be ignored in favor of closing the opening.
+
+### Custom Field Object Parameters
```

Comment:
This is copy and pasted from the POST opening section, right? You could probably just link to it rather than duplicate it, no?

Reply:
I could, the only difference is the `delete_value` parameter isn't available available in the POST. We also copy paste this in other endpoints, so I was following that